src/udev/net/ethtool-util.h

```diff
@@ -71,7 +71,21 @@ typedef enum NetDevPort {
         _NET_DEV_PORT_INVALID = -1
 } NetDevPort;
 
+typedef enum NetDevAdvertise {
+        NET_DEV_ADVERTISE_10BASET_HALF             = 0x001,
+        NET_DEV_ADVERTISE_10BASET_FULL             = 0x002,
+        NET_DEV_ADVERTISE_100BASET_HALF            = 0x004,
+        NET_DEV_ADVERTISE_100BASET_FULL            = 0x008,
+        NET_DEV_ADVERTISE_1000BASET_HALF           = 0x010,
+        NET_DEV_ADVERTISE_1000BASET_FULL           = 0x020,
+        NET_DEV_ADVERTISE_1000BASEKX_FULL          = 0x20000,
+        _NET_DEV_ADVERTISE_MAX,
+        _NET_DEV_ADVERTISE_INVALID = -1,
+} NetDevAdvertise;
```

Comment:
I don't think we should define out values like this. Please base the RHS values on kernel constants, like
```C
NET_DEV_ADVERTISE_10BASET_HALF = 1 << ETHTOOL_LINK_MODE_10baseT_Half_BIT,
```